.wordlist.txt

```diff
@@ -0,0 +1,16 @@
+Docusaurus
```

Comment:
In case you find it helpful, [this] is my wordlist, try adding a few words as per your understanding.